functions/Connect-DbaInstance.ps1

```diff
@@ -18,6 +18,14 @@ function Connect-DbaInstance {
 
         To execute SQL commands, you can use $server.ConnectionContext.ExecuteReader($sql) or $server.Databases['master'].ExecuteNonQuery($sql)
 
+        On Nov 6, 2020 we added a complete new way of processing inside of this command, available via the config switch sql.connection.experimental.
```

Comment:
Reads easier if you reference the build version instead of the date with modules.